_translator-files/po/de/Getting-Started.po

```diff
@@ -154,20 +150,15 @@ msgstr "Lasse zuerst die anderen Teilnehmer wissen, wer du bist. Klick dazu auf
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:52
-#, fuzzy, no-wrap
-#| msgid ""
-#| "<figure>\n"
-#| "  <img src=\"{{site.url}}/assets/img/en-screenshots/settings-profile.png\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
-#| "  <figcaption>Let people know who you are</figcaption>\n"
-#| "</figure>\n"
+#, no-wrap
 msgid ""
 "<figure>\n"
 "  <img src=\"{% include img/en-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
 "  <figcaption>Let people know who you are</figcaption>\n"
 "</figure>\n"
 msgstr ""
 "<figure>\n"
-"  <img src=\"{{site.url}}/assets/img/de-screenshots/settings-profile.png\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
+"  <img src=\"{% include img/en-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
```

Comment:
The language code in `{% include img/en-screenshots/settings-profile.inc %}` (en) should be changed to 'de'.